src/index.html

```diff
@@ -5,8 +5,433 @@
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price_label">
+              Price:
+            </p>
+
+            <p class="card__price_digit">
+              $2,199
+            </p>
+          </div>
+
+          <a
+            data-qa="card-hover"
+            href="/"
+            class="card__button_link"
+          >
+            buy
+          </a>
+        </div>
+
+        <div class="catalog__card card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="imac"
+          >
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product_code">
+            Product code: 195434
+          </p>
+
+          <div class="card__reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__reviews_count">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price_label">
+              Price:
+            </p>
+
+            <p class="card__price_digit">
+              $2,199
+            </p>
+          </div>
+
+          <a
+            href="/"
+            class="card__button_link"
+          >
+            buy
+          </a>
+        </div>
+
+        <div class="catalog__card card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="imac"
+          >
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product_code">
+            Product code: 195434
+          </p>
+
+
+          <div class="card__reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__reviews_count">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price_label">
+              Price:
+            </p>
+
+            <p class="card__price_digit">
+              $2,199
+            </p>
+          </div>
+
+          <a
+            href="/"
+            class="card__button_link"
+          >
+            buy
+          </a>
+        </div>
+
+        <div class="catalog__card card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="imac"
```

Comment:
alt attribute should be more descriptive. In this case you may just duplicate name of the item shown on picture, APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)